src/scheduler/job_info.cpp

```diff
@@ -1051,6 +1051,9 @@ query_jobs(status *policy, int pbs_sd, queue_info *qinfo, resource_resv **pjobs,
 
 		if (tdata->error || tdata->oarr == NULL) {
 			pbs_statfree(jobs);
+			if (tdata->oarr != NULL)
```

Comment:
You don't _need_ this if statement, but it's ok

Reply:
That's true, I'll remove it.